src/Mvc/Mvc.Core/src/Infrastructure/DefaultProblemDetailsFactory.cs

```diff
@@ -95,6 +108,6 @@ private void ApplyProblemDetailsDefaults(HttpContext httpContext, ProblemDetails
             problemDetails.Extensions["traceId"] = traceId;
         }
 
-        _configure?.Invoke(new() { HttpContext = httpContext!, ProblemDetails = problemDetails });
+        _configure?.Invoke(new() {HttpContext = httpContext!, ProblemDetails = problemDetails});
```

Comment:
Nit: our style guidelines recommend spacing between braces here.

```suggestion
        _configure?.Invoke(new() { HttpContext = httpContext!, ProblemDetails = problemDetails });
```